gtsfm/common/gtsfm_data.py

```diff
@@ -306,10 +306,18 @@ def get_avg_scene_reprojection_error(self) -> float:
     def log_scene_reprojection_error_stats(self) -> None:
         """Logs reprojection error stats for all 3d points in the entire scene."""
         scene_reproj_errors = self.get_scene_reprojection_errors()
-        logger.info("Min scene reproj error: %.3f", np.nanmin(scene_reproj_errors))
-        logger.info("Avg scene reproj error: %.3f", np.nanmean(scene_reproj_errors))
-        logger.info("Median scene reproj error: %.3f", np.nanmedian(scene_reproj_errors))
-        logger.info("Max scene reproj error: %.3f", np.nanmax(scene_reproj_errors))
+        logger.info(
+            "Min scene reproj error: %.3f", np.nanmin(scene_reproj_errors) if len(scene_reproj_errors) else np.NaN
+        )
+        logger.info(
+            "Avg scene reproj error: %.3f", np.nanmean(scene_reproj_errors) if len(scene_reproj_errors) else np.NaN
+        )
+        logger.info(
+            "Median scene reproj error: %.3f", np.nanmedian(scene_reproj_errors) if len(scene_reproj_errors) else np.NaN
+        )
+        logger.info(
+            "Max scene reproj error: %.3f", np.nanmax(scene_reproj_errors) if len(scene_reproj_errors) else np.NaN
+        )
```

Comment:
```
If len(scene_reproj_errors) is 0: 
   log.error("log_scene_reprojection_error_stats: Scene projection errors is empty!")
   return 
log as usual
```

Reply:
thanks. made the change

Reply:
@ayushbaid why would there be a case when the scene_reproj_errors would be empty or None? have we ever seen that? that would mean we have zero optimized points after filtering, which I have never seen